Exercise_A/index.html

```diff
@@ -10,6 +10,9 @@
   </head>
   <body>
     <!-- CODE HERE -->
+    <a href="file:///Users/BigAL/Documents/Ebook/IS/Web/Xcce/FTWD_Web_Day_02/Exercise_A/about.html">
+      <img src="file:///Users/BigAL/Documents/Ebook/IS/Web/Xcce/FTWD_Web_Day_02/assets/Liveserver.png" alt="link">
+    </a>
```

Comment:
Instead of giving a full path, give a path from where you HTML file is located. For example

 
```
<a href="about.html">
      <img src="assets/Liveserver.png" alt="link">
    </a>
```